src/engine/sync/syncable.h

```diff
@@ -8,13 +8,13 @@ enum SyncMode {
     SYNC_INVALID = -1,
     SYNC_NONE = 0,
     SYNC_FOLLOWER = 1,
-    // SYNC_MASTER_SOFT is a master that Mixxx has chosen automatically.
-    // depending on how decks stop and start, it may reassign soft master at will.
-    SYNC_MASTER_SOFT = 2,
-    // SYNC_MASTER_EXPLICIT represents an explicit request that the synacable be
-    // master. Mixxx will only remove a SYNC_MASTER_SOFT if the track is stopped or
+    // SYNC_LEADER_SOFT is a leader that Mixxx has chosen automatically.
```

Comment:
If you're renaming these anyway, could you also just make the enum a a scoped enum class and use Pascal Case for the members? Ideally, do that in a follow-up PR if possible.

Reply:
done